### ISSUE TITLE
[Feature Request] Showing whitespace

### ISSUE THREAD

**Issue by tillchen** [BoostIO/Boostnote#3046]
Hi:)

I'd like to see the feature of showing whitespaces as · like in VSCode or in Vim. It's very useful to manage trailing whitespaces and other stuff.

Thank you!

**Comment by AWolf81** [comment]
Good idea and it should be possible to add as an option to enable it.

But I think it's not that easy as it sounds. Maybe we can use some code from this [PR](https://github.com/sindresorhus/refined-github/pull/2073).

They're doing it with JS. Maybe it would be possible to create an addon for Codemirror similar to [trailingWhitespace addon](https://codemirror.net/addon/edit/trailingspace.js).

If we would only add trailing whitespace we could use the addon with some CSS but I would also add the other whitespace highlighting like dots for spaces and new line character.

**Comment by tillchen** [comment]
@AWolf81 Thank you:)

**Comment by Flexo013** [comment]
You probably want this feature for snippet notes (and potentially code blocks).

But just FYI, it already is possible to manage trailing whitespaces in markdown notes using the newly added markdown linter. If you enable the specific rule, [as found here](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md009---trailing-spaces), then you will receive a warning about trailing whitespaces, like so:
![image](https://user-images.githubusercontent.com/4507015/58698119-fd563b80-839a-11e9-8e11-e524f8807adc.png)

**Comment by tillchen** [comment]
@Flexo013  Yes, I'm aware of the linter feature. However, I think showing whitespaces is still a nice feature to add:) Thanks!
